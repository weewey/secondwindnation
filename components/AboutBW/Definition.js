import React from 'react';
import Typography from 'material-ui/Typography';
import { withStyles } from 'material-ui/styles';
import PropTypes from 'prop-types';

const styleSheet = theme => ({
  root: {
    width: '100%',
    height: '50vh',
    maxHeight: 300,
    textAlign: 'center',
    // backgroundColor: 'pink',
    justifyContent: 'center',
    display: 'flex',
    alignItems: 'center',
  },
  para: {
    width: '40em',
    padding: 20,
    margin: 'auto',
  },
  text: {
    marginTop: 16,
    fontWeight: 100,
  },
  title: {
    // fontWeight: 500,
    color: 'black',
  },
});

function Definition(props) {
  const classes = props.classes;
  return (
    <div className={classes.root}>
      <div className={classes.para}>
        <Typography type="display3" className={classes.title}>SECOND WIND</Typography>
        <Typography type="headline" className={classes.text}>A renewed energy for a fresh breakthrough.</Typography>
      </div>
    </div>
  );
}


Definition.propTypes = {
  classes: PropTypes.object.isRequired,
};

export default withStyles(styleSheet)(Definition);
