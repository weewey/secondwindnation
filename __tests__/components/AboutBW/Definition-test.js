import React from 'react';
import renderer from 'react-test-renderer';

import Definition from '../../../components/AboutBW/Definition';

describe('Definition component', () => {
  it('should render', () => {
    const definition = renderer.create(<Definition />,);
    const tree = definition.toJSON();
    expect(tree).toMatchSnapshot();
  },
);
});
