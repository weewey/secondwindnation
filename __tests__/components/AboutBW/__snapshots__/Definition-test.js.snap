// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Definition component should render 1`] = `
<div
  className="Definition-root-1"
>
  <div
    className="Definition-para-2"
  >
    <h1
      className="MuiTypography-root-5 MuiTypography-display3-7 Definition-title-4"
    >
      SECOND WIND
    </h1>
    <h1
      className="MuiTypography-root-5 MuiTypography-headline-10 Definition-text-3"
    >
      A renewed energy for a fresh breakthrough.
    </h1>
  </div>
</div>
`;
